src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -150,6 +163,14 @@ public TimestampInterval processTimeIntervalArgs ( Arguments args )
         if ( isTimeInterval(arg) )
         {
             end = time(null);
+
+            if ( round )
+            {
+                long modulu_this;
+                timeUnit(arg, modulu_this);
```

Comment:
i'm not sure what `modulu_this` is.. is it a typo?

Reply:
Fixed